### PR TITLE
Implement View Point Cloud Data Offline sample

### PR DESCRIPTION
**Pull request by philium**
This sample implementation includes a new mechanism for download portal data if needed as part of the build process.

### CODE REVIEW

**Comment by philium** [comment]
The initial build of this branch should take 30-40 seconds longer than normal while it downloads the necessary data. Please let me know if you experiences any problems.

**Comment by philium** [comment]
@pgruenler @saratchandrakarumuri Please expedite review if you can. Other sample implementations are waiting on changes in this PR.

**Comment by pgruenler** [comment]
I'm assigning the code review to @sbaskaran; she'll be done with her update 5 work sooner than I will.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ The ```master``` branch of this repository contains samples configured for the l
 1. Fork and then clone the repo. Don't know how? [Get started here.](http://htmlpreview.github.com/?https://github.com/Esri/esri.github.com/blob/master/help/esri-getting-to-know-github.html)
 2. Build and run the project to create a single app containing all of the samples.
 
+## Sample Data
+
+Some sample data is too large to store in the repository, so it is automatically downloaded at build time. The first time the app is built, a build script downloads the necessary data to `Portal Data`. The script only downloads data files that do not already exist, so subsequent builds will take significantly less time.
```
Comment (sbaskaran) [inline README.md]: @philium how is this different from the existing way of downloading files on the fly? For eg, "Offline Edit and Sync" sample among others.

Comment (philium) [reply README.md]: @sbaskaran The tile package used by the Offline Edit and Sync sample is stored in the repository, so it is downloaded as part of cloning the repository. The scene layer package used in this sample is too large to store in the repository, so the resource is missing when the repository is first cloned and only downloaded once the app is first built. Both are referenced in the Xcode project, are included in the app as resources, and are used as On Demand Resources at run time.

**Comment by philium** [comment]
@sbaskaran Thank you!

**Pull request merged**